widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/imageattachment/ImageAttachmentViewHolder.java

```diff
@@ -48,14 +48,14 @@ public ImageAttachmentViewHolder(
     }
 
     public void bind(AttachmentFile attachmentFile) {
-        String imageName = FileHelper.getFileName(attachmentFile);
+        String imageName = FilesKt.getFileName(attachmentFile);
```

Comment:
Not critical. Personally, I'm okay with `FileHelper` name. 🙂 I like it more than `FilesKt`.
`FileHelper` says that it contains some helper functionality.
`Files` can mean that it contains some files.
What is the purpose of "Kt" suffix? Does it mean that it can only be used from Kotlin code?

Reply:
You can ignore this comment unless anyone else also thinks so. Not critical at all.

Reply:
Kotlin also has such files, `kotlin.Tuples.kt`, `kotlin.collections.Collections.kt` so I'll let it as it is and wait for additional comments

Reply:
Okay, that makes sense, I don't mind.